landing_page/templates/landing_page/index.html

```diff
@@ -6,6 +6,11 @@ <h1>Welcome to ReeldIn</h1>
 <div class="content_div">
   <div id="title_div">
     <h2>Latest Recommendations...</h2>
+    <a href="{% url 'recommendations:index' %}">
+      <button id="get_recommendations_btm">
+        Get Recommendations
+      </button>
+    </a>
```

Comment:
This button was moved to the header in base.html, specifically inside profile_div. So it can be removed from this location.